components/financial_assistance/db/seedfiles/translations/en/dc/financial_assistance.rb

```diff
@@ -287,6 +289,19 @@
   "en.faa.income_temporary_message" => "<ul><li><strong>Start Date: </strong>If this is income you currently have, you must enter the date you began receiving this income in the 'From' field, or 1/1/%{current_year} (if the income started before this year). Do not enter a date in the future (for example, 1/1/%{next_year}) unless this is income that has not started but you expect to have in the future.</li><li><strong>End Date: </strong>Leave the 'To' field empty, unless you expect your income to end on a certain date (for example, if you have a seasonal job). Do not enter the last day of the year unless you know that you will not have this income the following year.</li></ul>",
   "en.faa.start_date_warning" => "<strong>Start Date:</strong> This is a required field. If you began receiving this income before this year, enter January 1 of the current year or the income start date. Only enter a future start date if you don't receive this income right now, but expect to receive it in the future.",
   "en.faa.end_date_warning" => "<strong>End Date:</strong> Leave the “%{end_date_field_label}” field blank unless your income will end on a specific date (e.g. the date your seasonal job ends). Only enter the last day of the current year if you won't have this income next year.",
+  "en.faa.incomes.job_income_note" => "Note: For job income this person currently receives, do not enter an end date into the ‘To’ field. Only enter an end date if the job income ended.",
+  "en.faa.incomes.from_employer" => "Does this person have income from an employer (wages, tips, bonuses, etc.) in %{assistance_year}?",
+  "en.faa.incomes.job_income" => 'Job Income',
+  "en.faa.incomes.employer" => 'Employer Name',
+  "en.faa.incomes.employer_phone" => 'Phone Number',
+  "en.faa.incomes.remove_all" => 'Are you sure you want to remove all job income(s)?',
+  "en.faa.incomes.confirm_deletion" => 'Confirm Deletion',
+  "en.faa.incomes.job_income_message" => "Tell us about job income for %{subject}, select “Continue to Next Step” when finished.",
+  "en.faa.incomes.job_end_date_note" => "Note, for %{income_type} income %{subject} currently receives, do not enter an end date into the “End Date” field, only enter an end date if the %{income_type} income has ended.",
+  "en.faa.incomes.from_employer_question" => "Does %{subject} have income from an employer?",
+  "en.faa.incomes.end_date_warning" => "Leave the “%{end_date_field_label}” field blank unless your income will end on a specific date (e.g. the date your seasonal job ends). Only enter the last day of the current year if you won't have this income next year.",
+  "en.faa.incomes.end_date_header" => "End Date:",
+  "en.faa.end_date_warning" => "<strong>End Date:</strong> Leave the 'To' field blank unless your income will end on a specific date (e.g. the date your seasonal job ends). Only enter the last day of the current year if you won't have this income next year.",
```

Comment:
is this string needed? We already have the `faa.incomes.end_date_warning` translation which should work for our reqs